instrumentation/opentelemetry-instrumentation-boto/setup.cfg

```diff
@@ -41,7 +41,6 @@ install_requires =
     opentelemetry-api ~= 1.3
     opentelemetry-semantic-conventions == 0.24b0
     opentelemetry-instrumentation == 0.24b0
-    opentelemetry-instrumentation-botocore == 0.24b0
```

Comment:
Hm, I'm not sure. Even if that is the case, do we want that to happen? It seems like a very problematic way of adding spans, just because some dependency is installed by an instrumentation package, imagine debugging that.

Reply:
I think it is reasonable to do. Otel Node does it quite a bit IIRC. Debugging would be the same whether it is installed automatically or manually but with automatic install, we ship a much better default UX IMO.